src/components/dev-dot-content/dev-dot-content.module.css

```diff
@@ -17,27 +17,38 @@
       display: none;
     }
 
-    & .alert.alert-info {
-      background-color: #f2f8ff;
-      border: 1px solid #cce3f3;
-      border-radius: 6px;
-      color: var(--token-color-palette-neutral-600);
-      padding: 16px;
-      margin: 20px 0;
-
-      & code {
-        background-color: white;
-        border: 1px solid #cce3fe;
-        border-radius: 5px;
+    /* Alert overrides. Note that alert classes are applied at content
+       extraction time, so it is currently difficult to remove or modify these
+       classes and their associated styles rather than override them */
+    & .alert {
+      /* Override letter-spacing, which in global styles is tracked out */
+      &.g-type-body {
+        letter-spacing: 0;
```

Comment:
This `.g-type-body` is the only thing I've added.

I've changed the nesting level here for clear nested selectors on `.alert`, which is causing the messy diff - apologies for the thrash here 🙇‍♂️